scalableminds/webknossos
# Fix volume tasks from file without specified Bbox

### URL of deployed dev instance (used for testing):
- https://volumetaskbbox.webknossos.xyz

### Steps to test:
- Create explorational volume, brush some, download
- Create volume task type
- Create task, select the volume task type, select “from NML”, select the downloaded zip
- As user, fetch the task, previously brushed data should be present

### Issues:
- fixes #5361

------
